plugins/flp_import/CMakeLists.txt

```diff
@@ -3,4 +3,8 @@ INCLUDE(BuildPlugin)
 INCLUDE_DIRECTORIES(unrtf)
 
 ADD_DEFINITIONS(--std=c++0x)
+IF(HOMEBREW)
+	ADD_DEFINITIONS(-stdlib=libc++)
```

Comment:
Can you try adding that option to line 5, i.e. always enabled? AFAIK it shouldn't cause any problems, only increase compatibility. (The same in `src/CMakeLists.txt`)